tests/UI/campaigns/regression/checkout/100PercentDiscount_FO.ts

```diff
@@ -202,7 +197,7 @@ describe('Regression - Checkout: Create 100% discount with free shipping discoun
         baseContext,
       );
 
-      await cartPage.addPromoCode(page, percentCartRule.code);
+      await cartPage.addPromoCode(page, percentCartRule.code as string);
```

Comment:
Do we have any other way? I didn't see too much cast since the beginning of the migration, if we can avoid it it would be way better as it can lead to side effects and wrong type flows